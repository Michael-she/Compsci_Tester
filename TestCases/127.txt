d a 3 4
d a 4 3
l a 5 5
#
5 5 r
5 6 d
3 4 d
2 4 l
4 6 l
4 5 l
4 3 d
3 3 l
4 4 d
3 4 d
2 3 d
1 3 d
2 4 l
2 3 l
3 2 l
3 1 l
2 2 d
1 2 l
0 3 l
0 2 l
1 1 l
1 0 d
3 0 d
2 0 d
