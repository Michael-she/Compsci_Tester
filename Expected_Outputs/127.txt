pygame 2.5.2 (SDL 2.28.3, Python 3.8.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  | a|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  | A|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  | a|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  | A|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  | A|  |  | a|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  | A|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  | A|  |  | a|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  | A|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  | A|  |  | a|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  | A|  | a|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  | A| a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  | a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  | a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  | A|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  | A|  | a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  | A|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | A| a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  | A|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  | a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  | A|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  | a|  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  | A|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  | a|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  | A|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  | a|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  | A|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  | a|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  | a|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  | a|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  | a|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | A|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  | a|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  | A|  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  | a|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  | A|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 | a|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  | A|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 | a| A|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 | a| A|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 | A|  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 | a| A|  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
Light loses
